src/Translator/assets/test/translator.test.ts

```diff
@@ -387,6 +388,27 @@ describe('Translator', () => {
             );
         });
 
+        test('missing message should throw an error if `throwWhenNotFound` is true', () => {
+            throwWhenNotFound(true);
+            setLocale('fr');
+
+            const MESSAGE_IN_ANOTHER_DOMAIN: Message<{ security: { parameters: NoParametersType } }, 'en'> = {
+                id: 'Invalid credentials.',
+                translations: {
+                    messages: {
+                        en: 'Invalid credentials.',
+                    },
+                },
+            };
+
+            expect(() => {
+                trans(MESSAGE_IN_ANOTHER_DOMAIN);
+            }).toThrow(`No translation message with id "${MESSAGE_IN_ANOTHER_DOMAIN.id}" is found.`);
```

Comment:
Prefer to inline strings in tests, this way we are sure about what they contains:

```suggestion
            }).toThrow(`No translation message with id "Invalid credentials." is found.`);
```